crates/next-core/src/next_font_google/mod.rs

```diff
@@ -389,7 +383,7 @@ async fn font_options_from_query_map(query: QueryMapVc) -> Result<NextFontGoogle
         };
 
     self::options::options_from_request(&parse_json_with_source_context(json)?, &FONT_DATA)
-        .map(NextFontGoogleOptionsVc::new)
+        .map(NextFontGoogleOptionsVc::cell)
```

Comment:
This still has the same bug. Every time this is called, we're creating a new instance of the cell. If the input value is expected to be equivalent, we want to implement a `#[turbo_tasks::function]fn new(opts: NextFontGoogleOptions) -> NextFontGoogleOptionsVc {}